applications/trilinos_application/custom_utilities/trilinos_cutting_app.h

```diff
@@ -257,13 +257,13 @@ class TrilinosCuttingApplication
 
         int ierr = -1;
         ierr = aux_non_overlapping_graph->GlobalAssemble(Insert,true); //Epetra_CombineMode mode=Add);
-        if (ierr < 0) KRATOS_THROW_ERROR(std::logic_error, "epetra failure --> ln 249", "");
+        KRATOS_ERROR_IF(ierr < 0) << "epetra failure --> " << __LINE__ << std::endl;
```

Comment:
KRATOS ERROR macro has the __line__ incorporated so there are no need to re report it